clients/gtest/blas1/asum_gtest.cpp

```diff
@@ -62,49 +62,22 @@ namespace
         // Google Test name suffix based on parameters
         static std::string name_suffix(const Arguments& arg)
         {
-            HipBLAS_TestName<asum_test_template> name(arg.name);
-            name << hipblas_datatype2string(arg.a_type);
-
-            if(strstr(arg.function, "_bad_arg") != nullptr)
-            {
-                name << "_bad_arg";
-            }
-            else
-            {
-                constexpr bool is_batched = (BLAS1 == blas1::asum_batched);
-                constexpr bool is_strided = (BLAS1 == blas1::asum_strided_batched);
-
-                name << '_' << arg.N << '_' << arg.incx;
-
-                if(is_strided)
-                {
-                    name << '_' << arg.stride_x;
-                }
-
-                if(is_batched || is_strided)
-                {
-                    name << "_" << arg.batch_count;
-                }
-            }
-
-            if(arg.api == FORTRAN)
-            {
-                name << "_F";
-            }
-
-            return std::move(name);
+            std::string name;
+            testname_asum(arg, name);
+            return name;
```

Comment:
return std::move(name);
?  Did you find it didn't avoid a copy

Reply:
No, std::move should still be useful, just overlooked. Added back.